Assets/RGLUnityPlugin/Scripts/RadarModels/RadarConfiguration.cs

```diff
@@ -13,35 +13,54 @@
 // limitations under the License.
 
 using System;
+using System.Linq;
 using UnityEngine;
 
 namespace RGLUnityPlugin
 {
     [Serializable]
-    public class RadarConfiguration
+    public struct RadarParametersScope
     {
-        [Min(0.0f)] public float minRange;
-
-        [Min(0.0f)] public float maxRange;
+        [Tooltip("Beginning distance range for the parameters.")]
+        [Min(0.0f)] public float beginDistance;
+        [Tooltip("End range distance for the parameters")]
+        [Min(0.0f)] public float endDistance;
+        [Tooltip("Maximum distance difference to create a new radar detection")]
```

Comment:
```suggestion
        [Tooltip("Minimum distance difference to create separate detections.")]
```
In my opinion, similar suggestion refers to lines 30 and 32 below.